__tests__/app.test.js

```diff
@@ -58,17 +58,15 @@ describe('/api/articles/:article_id', () => {
         .get('/api/articles/1')
         .expect(200)
         .then(({body}) => {
-            expect(body.article).toHaveLength(1)
-            body.article.forEach((article) => {
-                expect(typeof article.title).toBe('string')
-                expect(typeof article.article_id).toBe('number')
-                expect(typeof article.author).toBe('string')
-                expect(typeof article.body).toBe('string')
-                expect(typeof article.topic).toBe('string')
-                expect(typeof article.created_at).toBe('string')
-                expect(typeof article.votes).toBe('number')
-                expect(typeof article.article_img_url).toBe('string')              
-            })
+            expect(body.article.article_id).toBe(1);
```

Comment:
I would not expect a single article to be in an array. This should be a single object.